scripts/create_data_extract.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import tile_pb2
+import random
+import json, gzip
+
+
+#Read in v0.1 geojson OSMLR tiles from AWS to generate segment speed files that 
+#ie. week0_2017/level/0/tileid/segment_speeds.json (same as directory structure output as OSMLR input)
+#https://s3.amazonaws.com/osmlr-tiles/v0.1/pbf/0/000/747.osmlr
+def loadOSMLR():
+  osmlr_tile = tile_pb2.Tile()
+  f = open(sys.argv[1], "rb")
+  osmlr_tile.ParseFromString(f.read())
+  f.close()
+  
+  return osmlr_tile
+
+
+#Iterates through the segments, listing the lengths
+def list_segment_lengths(osmlr_tile, level, tile_id):
+  id = 0
+  segments_dict = {}
+  for entry in osmlr_tile.entries:
+    length = 0
+    segment_id = (id << 25) | (tile_id << 3) | level
+    id += 1
+    if entry.HasField("segment"): 
+      for loc_ref in entry.segment.lrps:
+        if loc_ref.HasField('length'):
+          length = length + loc_ref.length
+        print segment_id, level, tile_id, id, length
```

Comment:
@dnesbitt61 , could something else be incorrect with how I'm getting the segment id from the manila data?  Or could something be wrong with the data that I'm using from your drive?  Lou is not seeing segment id 849766009720 in the 0/002/415.json data extract.

This is what I see at the expected index of 2532:

segmentId, level, tile, index, length
84926286712 0 2415 2532 92
84926286712 0 2415 2532 92
84959841144 0 2415 2533 263
84959841144 0 2415 2533 263
84993395576 0 2415 2534 263
84993395576 0 2415 2534 263